src/components/_tests_/ButtonPanel.test.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/forbid-foreign-prop-types */
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
```

Comment:
- [ ] It is a requirement for this project to use the @testing-library/react instead of the react-dom one, please change this wherever necessary to comply with the requirements.